plugins/modules/monitoring/datadog/datadog_event.py

```diff
@@ -90,6 +95,19 @@
     api_key: 9775a026f1ca7d1c6c5af9d94d9595a4
     app_key: j4JyCYfefWHhgFgiZUqRm63AXHNZQyPGBfJtAzmN
     tags: 'aa,bb,#host:{{ inventory_hostname }}'
+
+- name: Post an event with several tags to another endpoint
+  community.general.datadog_event:
+    title: Testing from ansible
+    text: Test
+    api_key: 9775a026f1ca7d1c6c5af9d94d9595a4
+    app_key: j4JyCYfefWHhgFgiZUqRm63AXHNZQyPGBfJtAzmN
+    api_host: 'https://example.datadoghq.eu'
```

Comment:
if the expected format is like that, maybe we should call it `api_url` instead of `_host`

Reply:
True, but then that would differ from what Datadog python lib officially calls it
https://github.com/DataDog/datadogpy/blob/master/datadog/__init__.py#L65

To avoid any confusion I'd rather use the same variable name used in their lib